RedHatInsights/patchman-ui
# chore(RHINENG-3148, RHINENG-5936): Upgrade React and PatternFly

Implements https://issues.redhat.com/browse/RHINENG-3148, https://issues.redhat.com/browse/RHINENG-5936.

This upgrades React to use version 18, PatternFly to use version 5, and other related packages that affected with this upgrade to avoid conflicting peer dependencies. This makes sure the unit and cypress test are running fine.

## How to test

Make sure the application is built both for development and production environment. Ensure you are able to run it locally, the components (tables, dropdown, modals, empty states, etc.) are not broken and visually are up-to-date with PF5 or stay the same.
